File: First assignment/C/2nd question.c
#include <stdio.h>

int main()
{
    int count = 0;
	double multiply = 1;
	int integer;
	printf("Enter the integer for which you want calculate factorial = ");
	scanf("%d", &integer);
	int backup = integer;
	if(integer < 0){
	    while(integer < 0){
		    printf("The integer must be > or = to 0");
	    	scanf("%d", &integer);
	    }
	}
	
	while(integer>0){
		multiply = multiply*integer;
		integer = integer-1;
	}
	printf("The factorial of %d is = %lf", backup, multiply);

    return 0;
}

File: First assignment/Python/2nd question.py
count = 0
multiply = 1
integer = int(input("Enter the integer for which you want calculate factorial = "))
if integer < 0:
    while integer < 0:
        integer = int(input("The integer must be > or = to 0 = "))
        backup = integer
while (integer > 0):
    multiply =multiply*integer
    integer=integer-1
print("The factorial of",backup,"is =",multiply)
